admin/sql/updates/20220328-mbs-12250-mirror.sql

```diff
@@ -1,6 +1,6 @@
 BEGIN;
 
-CREATE SCHEMA dbmirror2;
+CREATE SCHEMA IF NOT EXISTS dbmirror2 AUTHORIZATION musicbrainz;
```

Comment:
Does commit https://github.com/metabrainz/musicbrainz-server/pull/2523/commits/4d8d9c32d4c4350c228034de24b502c7d15645bd require `POSTGRES_USER` to be set to `musicbrainz` on mirrors?

Reply:
Yes, I think I should revert this.

Reply:
Reverted the `AUTHORIZATION` part so that the commit only adds `IF NOT EXISTS`. (The owner of the schema will default to the user running the script.)